pages/docs/prefetching.md

```diff
@@ -1,23 +1,29 @@
 # Prefetching Data
 
+## Top-Level Page Data
+
 There’re many ways to prefetch the data for SWR. For top level requests, [`rel="preload"`](https://developer.mozilla.org/en-US/docs/Web/HTML/Preloading_content) is highly recommended:
 
 ```html
 <link rel="preload" href="/api/data" as="fetch" crossorigin="anonymous">
 ```
 
-This will prefetch the data before the JavaScript starts downloading. And your incoming fetch requests will reuse the result (including SWR, of course).
+Just put it inside your HTML `<head>`. It’s easy, fast and native.
+
+It will prefetch the data as long as the HTML loads, even before the JavaScript starts to download. All your incoming fetch requests with the same URL will reuse the result (including SWR, of course).
```

Comment:
```suggestion
It will prefetch the data when the HTML loads, even before JavaScript starts to download. All your incoming fetch requests with the same URL will reuse the result (including SWR, of course).
```